partner_street_number/__openerp__.py

```diff
@@ -21,17 +21,18 @@
 {
     "name": "Street name and number",
     "summary": "Introduces separate fields for street name and street number.",
-    "version": "8.0.0.1.0",
+    "version": "9.0.0.1.0",
     "author": "Therp BV,Odoo Community Association (OCA)",
     "website": "https://github.com/oca/partner-contact",
     "category": 'Tools',
     "depends": [
-        'base'
-        ],
+        'web',
```

Comment:
The placement of the street name and number fields is now tweaked using CSS instead of using inline style, and the css link is inserted in an assets view from the web module.